api/src/test/java/org/openmrs/scheduler/SchedulerServiceTest.java

```diff
@@ -413,4 +414,22 @@ public void shouldSaveLastExecutionTime() throws Exception {
 		assertEquals("Last execution time in seconds is wrong", actualExecutionTime / 1000, td.getLastExecutionTime()
 		        .getTime() / 1000, 1);
 	}
+
+	/**
+	 * @see SchedulerService#getTaskByUuid(String)
+	 */
+	@Test
+	public void getTaskByUuid_shouldGetTaskByUuid() throws Exception {
+		TaskDefinition td = Context.getSchedulerService().getTaskByUuid("1365a6da-6493-4e9b-b950-5af1b392aaa3");
+		assertNotNull(td);
+	}
+
+	/**
+	 * @see SchedulerService#getTaskByUuid(String)
+	 */
```

Comment:
Do you think these are useful to the developer? testcase name itself has which method is under testing and class name AKA test suite has which class is under test. What do you think?

Reply:
yes its helpful to refer to the developer to the actual service method being tested